src/pages/index.js

```diff
@@ -12,6 +12,8 @@ const IndexPage = () => {
     // console.log({ osat })
     setIp("")
     if (osat.loppu === "" || osat.loppu.length !== 4) return
+    if (osat.loppu.slice(0, 1) !== "2" && osat.loppu.slice(0, 1) !== "3") return
+    if (parseInt(osat.loppu.slice(1)) >= 255 || parseInt(osat.loppu.slice(1)) === 0 || isNaN(parseInt(osat.loppu.slice(1)))) return
```

Comment:
This will make sure last 3 digits or characters are only valid to 254. In second it will not allow using 2000 or 3000 as id. In last part it will check if it's not integer but it will fail to work as intended when first character is number but second is letter, this should be avoided other way.